### PR TITLE
Remove non-purposefully added response headers, standardize response header formatting

### LINKED ISSUE #397: Remove X-Runtime and Server-Response-Time response headers from examples
We should take any response headers that aren't relevant or explicitly meaningful out of examples documented on the site

### PR DESCRIPTION
**Pull request by MaxPhilips**
Fixes #397

Description
----
Some response headers are auto-added by the server and may change at any time, so we should focus our documentation around response headers that are meaningful

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by MaxPhilips** [comment]
pre-merge verification

Screen Shot 2020-07-10 at 2 36 59 PM

File: content/millennium/dstu2/general-clinical/allergy-intolerance.md [hunk 1]
```diff
@@ -163,24 +163,22 @@ Create new allergies.
...
+X-Xss-Protection: 1; mode=block
+Pragma: no-cache
+X-Request-Id: b0fee21c20d2a240d9b4b86cfbcbd87c
+Etag: W/"6167733"
+X-Frame-Options: SAMEORIGIN
+X-Content-Type-Options: nosniff
+Expires: Mon, 01 Jan 1990 00:00:00 GMT
+Last-Modified: Tue, 28 Feb 2017 21:03:00 GMT
+Location: https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance/6167733
+Status: 201 Created
```
Comment (MaxPhilips) [inline content/millennium/dstu2/general-clinical/allergy-intolerance.md:179]: remove Status too

Comment (MaxPhilips) [reply content/millennium/dstu2/general-clinical/allergy-intolerance.md]: https://github.com/cerner/fhir.cerner.com/pull/425/commits/a9eafa767cd108f358f9167df598e997ee776e95

**Comment by MaxPhilips** [comment]
pre-merge verification

Screen Shot 2020-07-10 at 2 48 37 PM

**Comment by ebbp123** [comment]
Some files may need to be updated.
From main. These have some instances with the Status, Server-Responss-Time- and X-Runtime headers:
- https://github.com/cerner/fhir.cerner.com/blob/main/content/millennium/r4/documents/document-reference.md
- https://github.com/cerner/fhir.cerner.com/blob/main/content/millennium/r4/diagnostic/observation.md
- https://github.com/cerner/fhir.cerner.com/blob/main/content/millennium/r4/individuals/related-person.md
- https://github.com/cerner/fhir.cerner.com/blob/main/content/millennium/r4/medications/medication-request.md
- https://github.com/cerner/fhir.cerner.com/blob/main/content/millennium/r4/security/provenance.md

**Comment by MaxPhilips** [comment]
fixed for new resources with 029736c

**Review by ebbp123 — APPROVED** [review]

**Review by sydneanschutz — APPROVED** [review]

**Comment by MaxPhilips** [comment]
post-merge validation
Screen Shot 2021-02-01 at 4 39 25 PM

**Pull request merged**